e3sm_to_cmip/__main__.py

```diff
@@ -95,8 +95,7 @@ def main():
         tables_path=tables_path,
         var_list=var_list,
         freq=freq,
-        realm=realm,
-        simple=simple)
+        realm=realm)
 
```

Comment:
Awesome, thanks for fixing my oversight!

Reply:
no prob - almost gives the impression I know what I'm doing...